LICENSE

```diff
@@ -1,14 +1,42 @@
+This project is forked from rrweb-io/rrweb (https://github.com/rrweb-io/rrweb) under the MIT license:
+
+MIT License
+
+Copyright (c) 2018 Contributors (https://github.com/rrweb-io/rrweb/graphs/contributors) and SmartX Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+Our modifications to this project are also released under the MIT license:
+
 Copyright (c) 2023 Sentry (https://sentry.io) and individual contributors. All rights reserved.
```

Comment:
```suggestion
Copyright (c) 2023 Functional Software, Inc. dba Sentry
```

This is our [standard🔒](https://www.notion.so/sentry/Open-Source-Legal-Policy-ac4885d265cb4d7898a01c060b061e42?pvs=4#a3bea8a32a7d4de8ab252dc9fa2de2a3) copyright language. Looks like maybe you started with `sentry-javascript`'s `LICENSE`? I made https://github.com/getsentry/sentry-javascript/pull/8468 to bring that into line as well.